<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ThreadViewUI</class>
 <widget class="QWidget" name="ThreadViewUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>706</width>
    <height>364</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form1</string>
  </property>
  <layout class="QVBoxLayout">
   <property name="spacing">
    <number>6</number>
   </property>
   <property name="margin">
    <number>2</number>
   </property>
   <item>
    <layout class="QHBoxLayout">
     <property name="spacing">
      <number>6</number>
     </property>
     <property name="margin">
      <number>0</number>
     </property>
     <item>
      <widget class="QPushButton" name="mAddNoteButton">
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="threadview.qrc">
         <normaloff>:/threadview/message.png</normaloff>:/threadview/message.png</iconset>
       </property>
       <property name="default">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="mShowRecursiveButton">
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="threadview.qrc">
         <normaloff>:/threadview/tree_win.png</normaloff>:/threadview/tree_win.png</iconset>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="mShowLastCheck">
       <property name="text">
        <string>Last</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="mDaysCombo">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="minimumSize">
        <size>
         <width>60</width>
         <height>0</height>
        </size>
       </property>
       <property name="editable">
        <bool>true</bool>
       </property>
       <item>
        <property name="text">
         <string>5</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>10</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>20</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>50</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>100</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="textLabel1">
       <property name="text">
        <string>Days</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Expanding</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>130</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="textLabel6">
       <property name="text">
        <string>Show:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="mUnreadFilter">
       <item>
        <property name="text">
         <string>All Messages</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Unread Messages</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QSplitter" name="splitter6">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <widget class="RecordTreeView" name="mThreadView">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="alternatingRowColors">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QTextBrowser" name="mMessageLabel"/>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>RecordTreeView</class>
   <extends>QTreeView</extends>
   <header>recordtreeview.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>mThreadView</tabstop>
  <tabstop>mAddNoteButton</tabstop>
  <tabstop>mShowRecursiveButton</tabstop>
 </tabstops>
 <resources>
  <include location="threadview.qrc"/>
 </resources>
 <connections/>
</ui>
